### PR TITLE
fix: [oadp-1.1] OADP-2094 Always check restore resource priorities

### PR DESCRIPTION
**Pull request by mateusoliveira43**
Address Tiger's comment https://github.com/openshift/oadp-operator/pull/1132#discussion_r1303365262

### CODE REVIEW

File: controllers/velero.go [hunk 1]
```diff
@@ -394,6 +394,8 @@ func (r *DPAReconciler) customizeVeleroContainer(dpa *oadpv1alpha1.DataProtectio
 	}
 	// Append restic timeout option manually. Not configurable via install package, missing from podTemplateConfig struct. See: https://github.com/vmware-tanzu/velero/blob/8d57215ded1aa91cdea2cf091d60e072ce3f340f/pkg/install/deployment.go#L34-L45
 	veleroContainer.Args = append(veleroContainer.Args, fmt.Sprintf("--restic-timeout=%s", resticTimeout))
+	// TODO meaningful comment
```
Comment (mateusoliveira43) [inline controllers/velero.go:397]: if you are ok with the change, please help me with waht to comment here

File: pkg/common/common.go [hunk 2]
```diff
@@ -7,17 +7,18 @@ import (
...
+	VeleroNamespace                  = "oadp-operator"
+	OADPOperator                     = "oadp-operator"
+	OADPOperatorVelero               = "oadp-operator-velero"
+	DataMover                        = "volume-snapshot-mover"
+	DataMoverController              = "data-mover-controller"
+	DataMoverControllerContainer     = "data-mover-controller-container"
+	OADPOperatorServiceAccount       = "openshift-adp-controller-manager"
+	VolSyncDeploymentName            = "volsync-controller-manager"
+	VolSyncDeploymentNamespace       = "openshift-operators"
+	DefaultRestoreResourcePriorities = "securitycontextconstraints,customresourcedefinitions,namespaces,storageclasses,volumesnapshotclass.snapshot.storage.k8s.io,volumesnapshotcontents.snapshot.storage.k8s.io,volumesnapshots.snapshot.storage.k8s.io,datauploads.velero.io,persistentvolumes,persistentvolumeclaims,serviceaccounts,secrets,configmaps,limitranges,pods,replicasets.apps,clusterclasses.cluster.x-k8s.io,endpoints,services,-,clusterbootstraps.run.tanzu.vmware.com,clusters.cluster.x-k8s.io,clusterresourcesets.addons.cluster.x-k8s.io"
```
Comment (mateusoliveira43) [inline pkg/common/common.go:21]: after debugging with wes, we think there could be a typo/wrong order in here. What you think?

File: controllers/velero.go (hunk 1 above)
Comment (kaovilai) [reply controllers/velero.go:397]: Overriding velero restore resource priorities to OpenShift default (ie. SecurityContextConstraints needs to be restored before pod/SA)

Comment (mateusoliveira43) [reply controllers/velero.go:397]: right, thanks! going to add it in the next commit

File: pkg/common/common.go (hunk 2 above)
Comment (mateusoliveira43) [reply pkg/common/common.go]: Thanks for the help debugging @kaovilai !
Thanks @shubham-pampattiwar for finding the missing CR!

**Review by sseago — APPROVED** [review]

**Review by shubham-pampattiwar — APPROVED** [review]

**Pull request merged**